.build/run-ci.sh

```diff
@@ -0,0 +1,157 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# variables, with defaults
+[ "x${CASSANDRA_DIR}" != "x" ] || CASSANDRA_DIR="$(readlink -f $(dirname "$0")/..)"
+[ "x${KUBECONFIG}" != "x" ]    || KUBECONFIG="${HOME}/.kube/config"
+[ "x${KUBE_NS}" != "x" ]       || KUBE_NS="default" # FIXME – doesn't work in other namespaces :shrug:
+
+# pre-conditions
+command -v kubectl >/dev/null 2>&1 || { echo >&2 "kubectl needs to be installed"; exit 1; }
+command -v helm >/dev/null 2>&1 || { echo >&2 "helm needs to be installed"; exit 1; }
+
+# Parse command-line arguments
+while [[ $# -gt 0 ]]; do
+    key="$1"
+    case $key in
+        -c|--kubeconfig)
+            KUBECONFIG="$2"  # This sets the KUBECONFIG variable to the next argument.
+            shift            # This shifts the arguments to the left, discarding the current argument and moving to the next one.
+            shift            # This is an additional shift to move to the argument after the option value.
+            ;;
+        -ctx|--kubecontext)
+            unset KUBECONFIG
+            KUBECONTEXT="$2" # This sets the KUBECONTEXT variable to the next argument.
+            shift            # This shifts the arguments to the left, discarding the current argument and moving to the next one.
+            shift            # This is an additional shift to move to the argument after the option value.
+            ;;
+        --include-test-stage)
+            INCLUDE_TEST_STAGE="$2"
+            shift
+            shift
+            ;;
+        *)
+            echo "Unknown option: $1"
+            exit 1
+            ;;
+    esac
+done
+
+if [ -z "$KUBECONFIG" ] && [ -z "$KUBECONTEXT" ]; then
+    echo "Please provide either the path to the kubeconfig using -c|--kubeconfig option or the kubecontext using -ctx|--kubecontext option."
+    exit 1
+fi
+
+# This sets the kubeconfig and kubecontext if provided
+if [ -n "$KUBECONFIG" ]; then
+    export KUBECONFIG="$KUBECONFIG"
+fi
+
+if [ -n "$KUBECONTEXT" ]; then
+    kubectl config use-context "$KUBECONTEXT"
+fi
+
+if ! kubectl get namespace ${KUBE_NS} >/dev/null 2>/dev/null ; then
+    kubectl create namespace ${KUBE_NS}
+fi
+
+if [ -n "$INCLUDE_TEST_STAGE" ]; then
+    # The variable is not empty, so it has a value
+    echo "INCLUDE_TEST_STAGE is not empty. Its value is: $INCLUDE_TEST_STAGE"
+else
+    # The variable is empty, so assign a default value
+    INCLUDE_TEST_STAGE="lint,stress,fqltool"
+    echo "INCLUDE_TEST_STAGE is empty. Assigning default value: $INCLUDE_TEST_STAGE"
+fi
+
+
+TARGETS=".jenkins/job/DslJob.jenkins"
+REPO_BRANCH="infracloud/cassandra-5.0"
+REPO_URL="https://github.com/infracloudio/cassandra.git"
+
+
+sed -i -e "/targets:/s|:.*$|: \"$TARGETS\"|" \
+    -e "/repositoryBranch:/s|:.*$|: \"$REPO_BRANCH\"|" \
+    -e "/repositoryUrl:/s|:.*$|: \"$REPO_URL\"|" jenkins-deployment.yaml
+
```

Comment:
fix needed for macos compatibility
```suggestion
sed -e "/targets:/s|:.*$|: \"$TARGETS\"|" \
    -e "/repositoryBranch:/s|:.*$|: \"$REPO_BRANCH\"|" \
    -e "/repositoryUrl:/s|:.*$|: \"$REPO_URL\"|" ${CASSANDRA_DIR}/.build/jenkins-deployment.yaml > ${CASSANDRA_DIR}/build/jenkins-deployment.yaml

```